libraries/libraries_api.go

```diff
@@ -231,6 +231,10 @@ func (library Library) String() string {
 	return "unknown"
 }
 
+func (library Library) GetID(clusterId string) string {
+	return fmt.Sprintf("%s/%s", clusterId, library.String())
+}
+
```

Comment:
We need a test for it in the `libraries` package, otherwise the code coverage will be decreasing (Go's code coverage tool calculates code coverage per each package, so even if there is a usage in another package it's not taken into account)